pkg/oauthserver/help_test.go

```diff
@@ -0,0 +1,97 @@
+package oauthserver_test
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/openshift/oauth-server/pkg/audit"
+	apiaudit "k8s.io/apiserver/pkg/apis/audit"
+	kaudit "k8s.io/apiserver/pkg/audit"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+func withKeyValues(values ...[2]string) string {
+	const (
+		key   = 0
+		value = 1
+	)
+
+	form := url.Values{}
+
+	for _, tuple := range values {
+		form.Add(tuple[key], tuple[value])
+	}
+
+	return form.Encode()
+}
+
+func withKeyValue(key, value string) [2]string {
+	return [2]string{key, value}
+}
```

Comment:
remove completely, you're using this as a constant

Reply:
ultra true

Reply:
done